src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setQuery(value);
+  };
+
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    const sutib1 = movie.description.toLowerCase()
+      .includes(query.trim().toLowerCase());
```

Comment:
you transform the query several times in your code
let's create separate variable before the filter method with refactored query value, that we'll be able to use here

Reply:
done)